fairmq/nanomsg/FairMQTransportFactoryNN.cxx

```diff
@@ -63,7 +63,8 @@ FairMQPollerPtr FairMQTransportFactoryNN::CreatePoller(const FairMQSocket& cmdSo
 
 void FairMQTransportFactoryNN::Shutdown()
 {
-    nn_term();
+    // nn_term();
+    // see https://www.freelists.org/post/nanomsg/Getting-rid-of-nn-init-and-nn-term,8
 }
```

Comment:
In the tests I create and destroy multiple nn transports in series. Before this PR it worked because I did not call `Shutdown`. But I need to call it for shmem and for correctness.